test/cases/api.sh

```diff
@@ -413,7 +413,15 @@ function collectMetrics(){
                           --cert /etc/osbuild-composer/client-crt.pem \
                           https://localhost/metrics)
 
-    echo "$METRICS_OUTPUT" | grep "^image_builder_composer_total_compose_requests" | cut -f2 -d' '
+    COMPOSES1=$(echo "$METRICS_OUTPUT" | grep "^image_builder_composer_total_compose_requests" | cut -f2 -d' ')
+
+    METRICS_OUTPUT2=$(curl https://localhost:9090/metrics)
```

Comment:
Shouldn't the URL use `8008` port instead of `9090`? 🤔  I would expect that this test should fail, but it did not from what I remember, which is weird...

Reply:
Turns out `test` doesn't work when the output of the function is captured in a variable, so I created a new function just for the test.

Reply:
Alternatively, you could have just a simple `if` block and `exit 1` in case the condition is true... I wonder if we use `test` in the same way in other places of the test case 🤔

Reply:
It's now failing and for the right reason: `curl: (7) Failed to connect to localhost port 9090: Connection refused`, might be a bit hard to spot though.
In all cases, this is a short lived change and once prometheus is moved to the new port, this new test function will be removed.
I checked all other `*.sh` scripts and we don't have other cases of `test` inside function with the returned value saved in a variable.